src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -176,38 +176,69 @@ static bool _appendChildShape(SvgNode* node, Shape* shape, float vx, float vy, f
 }
 
 
-static void _applyComposition(Paint* paint, const SvgNode* node, float vx, float vy, float vw, float vh)
+static void _applyClipPathComposition(Paint* paint, const SvgNode* node, float vx, float vy, float vw, float vh)
```

Comment:
I think it can be handled with one methid.
like this
`static void _applyComposition(Paint* paint, SvgComposite& comp, Matrix* transform, CompositeMethod method, float vx, float vy, float vw, float vh)`
how about this?

Reply:
Please find the changes

Reply:
oh my opinion was to reduce duplicate code by passing method type and SvgComposite as parameters.
The patch is working good. so I'll merge this PR and refactor it later